File: LNPG_02_ESTRUTURAS_DE_CONTROLE/ATV_1/C/main.c
#include <stdio.h>
#include <string.h>

int checkRG(const char *rg)
{
	if (strcmp(rg, "RG") == 0)
	{
		return 1;
	}
	else
	{
		return 0;
	}
}

int checkTicket(const char *ticket)
{
	if (strcmp(ticket, "Passagem") == 0)
	{
		return 1;
	}
	else
	{
		return 0;
	}
}

int checkBirthDate(const char *ticketDate, const char *rgDate)
{
	if (strcmp(ticketDate, rgDate) == 0)
	{
		return 1;
	}
	else
	{
		return 0;
	}
}

int main()
{
	int passengers;
	char rg[11], rgDate[11], ticket[11], ticketDate[11], seat[4];

	printf("Quantidade de passageiros: ");
	scanf("%d", &passengers);

	for (int i = 1; i <= passengers; i++)
	{
		printf("\nPassageiro %d", i);

		printf("\nRG? (Se sim, RG; Se não, Nao possui): ");
		scanf(" %[^\n]", rg);

		printf("Data de nascimento no RG (DD/MM/AAAA): ");
		scanf(" %[^\n]", rgDate);

		printf("Passagem? (Se sim, Passagem; Se não, Nao possui): ");
		scanf(" %[^\n]", ticket);

		printf("Data de nascimento na passagem (DD/MM/AAAA): ");
		scanf(" %[^\n]", ticketDate);

		printf("Assento (A12): ");
		scanf(" %[^\n]", seat);

		if (checkRG(rg) == 0)
		{
			printf("A saida e nessa direcao");
		}
		else if (checkTicket(ticket) == 0)
		{
			printf("A recepção é nessa direção!");
		}
		else if (checkBirthDate(ticketDate, rgDate) == 0)
		{
			printf("190!");
		}
		else
		{
			printf("O seu assento é %s!", seat);
		}
	}

	return 0;
}

File: LNPG_02_ESTRUTURAS_DE_CONTROLE/ATV_1/JAVA/Main.java
package ATV_1.JAVA;

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        Airport air = new Airport();

        System.out.print("Quantidade de passageiros: ");
        int passengers = Integer.parseInt(scan.nextLine());

        for (int i = 1; i <= passengers; i++) {
            System.out.println("Passageiro " + i);

            System.out.print("RG? (Se sim, RG; Se não, Nao possui): ");
            String rg = scan.nextLine();

            System.out.print("Data de nascimento no RG (DD/MM/AAAA): ");
            String rgDate = scan.nextLine();

            System.out.print("Passagem? (Se sim, Passagem; Se não, Nao possui): ");
            String ticket = scan.nextLine();

            System.out.print("Data de nascimento na passagem (DD/MM/AAAA): ");
            String ticketDate = scan.nextLine();

            System.out.print("Assento (A12): ");
            String seat = scan.nextLine();

            if (air.checkRG(rg) == false) {
                System.out.println("A saída é nessa direção!");
            } else if (air.checkTicket(ticket) == false) {
                System.out.println("A recepção é nessa direção!");
            } else if (air.checkBirthDate(ticketDate, rgDate) == false) {
                System.out.println("190!");
            } else {
                System.out.printf("O seu assento é %s!", seat);
            }
        }
        scan.close();
    }
}